docs/source/concepts/workflows.rst

```diff
@@ -156,25 +156,25 @@ The resulting provenance would look like the following:
 .. _fig_work_chains_provenance_add_multiply_workchain_logical:
 .. figure:: include/images/add_multiply_workchain_logical.png
 
-    The provenance generated by the work chain example that computers the sum and product directly in its outline steps instead of delegating it to calculation functions.
+    The provenance generated by the work chain example that computes the sum and product directly in its outline steps, instead of delegating it to calculation functions
 
 Note how, in contrast with the provenance of the previous correct solution from :numref:`fig_work_chains_provenance_add_multiply_workchain_full`, there are no explicit calculation nodes representing the computation of the sum and the product.
 Instead, all that computation is abstracted and represented by the single workflow node that represents the execution of the work chain.
 The logic inside of those outline steps is then 'hidden' or 'encapsulated' in the provenance graph by a single workflow node.
-Additionally, the output node representing the final product, only has a ``return`` link, even though it was 'created' by the work chain.
-This is because :ref:`workflow processes do not have the capacity to create new nodes<working_workfunctions_returning_data>`, and therefore in a sense in this example, the data provenance is lost.
+Additionally, the output node representing the final product, only has a ``return`` link, even though it was "*created*" by the work chain.
```

Comment:
before you changed to *created*, let's stick to that

Reply:
For this particular one, I just didn't want people to, in any shape or form, think that a work chain can *create* - but maybe it looks better overall without the quotation marks, and the readers will be able to understand just fine?

Reply:
I have removed the quotations.